ex03.exs

```diff
@@ -51,15 +51,33 @@ defmodule Ex03 do
   """
 
   def pmap(collection, process_count, function) do
-    # your code goes here
-    # I'm hoping to see a simple pipeline as the body of this function...
+    collection
+    |> getChunks(process_count)
+    |> createProcesses(function)
+    |> mergeProcesses()
+ end
+
+  def getChunks(collection, process_count) do
+    size = div Enum.count(collection),process_count
```

Comment:
This `div` returns the wrong value. Try it with 10 and 4. It says you want a chunk size of 2, but you really need 3.

It should be 

    div Enum.count(collection) + process_count - 1,process_count

# -2